rr_endline_detection/src/endline_detection.cpp

```diff
@@ -0,0 +1,108 @@
+#include "endline_detection.hpp"
```

Comment:
include a file descriptor as well describe the inputs and outputs of the node like the description in the path planner node cpp file

Reply:
done

Reply:
actually done